Hash/Hash.cs

```diff
@@ -0,0 +1,158 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FunctionHash
+{
+    public class Result

```

Comment:
Could you split it into standalone class files?

Reply:
I don't know if this code is needed. Hypothetically, it may be needed by those who will convert their answer to json. But okay, yes, it should be a separate file.